test/create-placeholders-for-phi-operands.ll

```diff
@@ -0,0 +1,153 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv -spirv-ext=+SPV_INTEL_variable_length_array
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis %t.rev.bc -o %t.rev.ll
+; RUN: FileCheck %s --input-file %t.rev.ll --check-prefix CHECK-LLVM
+
+; CHECK-LLVM: BB.1:
```

Comment:
Please add check for the Phi instruction. You should capture values in this instruction and check for them later.
```suggestion
; CHECK-LLVM: phi i8* [ [[savedstack:%.*]], {{.*}} ], [ [[savedstack:%.*]], {{.*}} ]
```